src/gui/sdl_mapper.cpp

```diff
@@ -919,7 +930,8 @@ class CStickBindGroup : public CBindGroup {
 	uint8_t emulated_buttons = 0;
 	int hats = 0;
 	int emulated_hats = 0;
-	int stick;
+	int stick_id{-1};    // Instance ID of the joystick as it appears in SDL events
+	int stick_index{-1}; // Index of the joystick in the system
```

Comment:
Please put the comments on the previous line. I really dislike long inline comments, and kcgen is the same. We eradicate existing ones on-sight, so please don't add new ones 😄